compiler/src/iree/compiler/Codegen/LLVMGPU/KernelConfig.cpp

```diff
@@ -573,6 +574,177 @@ setMatmulVectorDistributionConfig(IREE::GPU::TargetAttr target,
                                                targetSubgroupSize, configDict);
 }
 
+static LogicalResult
+setAttentionVectorDistributionConfig(IREE::GPU::TargetAttr target,
```

Comment:
Nice! this is clinical! btw do we just let K1 go through GPUTileReductionPass? 
If yes, are we ensuring that it will select serial reduction tile size that is aligned with our intrinsics.

Reply:
We cannot tile K1 for attention. Since we get our hueristics from the second matmul, our serial tile sizes are always aligned with our intrinsics.

Reply:
Nice nice :)